pages/k8s/install-local.md

```diff
@@ -135,11 +135,24 @@ All that remains is to deploy **Charmed Kubernetes**. A simple install can be ac
 juju deploy charmed-kubernetes
 ```
 
+
 This will install the latest stable version of **Charmed Kubernetes** with
 the default components and configuration. If you wish to customise this install
 (which may be helpful if you are close to the system requirements), please see
 the [main install page][install].
 
+<div class="p-notification--warning"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Bug Warning:</span>
+There is currently a bug, <a href="https://bugs.launchpad.net/charm-kubernetes-worker/+bug/1903566"> LP#1903566</a>,
+which prevents <code>Kubelet</code> from running properly on LXD. Until this is fixed, a 
+workaround is to configure kubelet to protect the kernel defaults:
```

Comment:
Other way around, setting this disables kernel default protection. I would rewrite this to say something like:

```
Until this is fixed, a workaround is to configure kubelet to disable kernel default protection:
```

or

```
Until this is fixed, a workaround is to configure kubelet to override kernel defaults:
```

Reply:
ah, okay. thanks!